MultiVersions/Forge/main/integrateddynamics/lang/zh_tw.json

```
@@ -0,0 +1,1873 @@
+    "block.integrateddynamics.proxy.info": "將變數公開作為項目使用。在合成台上清空。",
+    "block.integrateddynamics.materializer": "提取機",
+    "block.integrateddynamics.materializer.info": "將變數的值複製到另一個變數卡上",
+    "block.integrateddynamics.menril_log": "門瑞歐木原木",
+    "block.integrateddynamics.menril_log_filled": "飽滿的門瑞歐木原木",
+    "block.integrateddynamics.menril_log_stripped": "剝皮門瑞歐木原木",
+    "block.integrateddynamics.menril_leaves": "門瑞歐木樹葉",
+    "block.integrateddynamics.menril_sapling": "門瑞歐木樹苗",
+    "block.integrateddynamics.menril_planks": "門瑞歐木材",
+    "block.integrateddynamics.menril_slab": "門瑞歐木半磚",
+    "block.integrateddynamics.menril_fence": "門瑞歐木柵欄",
+    "block.integrateddynamics.menril_fence_gate": "門瑞歐木柵欄門",
+    "block.integrateddynamics.menril_wood": "門瑞歐木塊",
+    "block.integrateddynamics.menril_wood_stripped": "剝皮門瑞歐木塊",
+    "block.integrateddynamics.crystalized_menril_block": "門瑞歐結晶方塊",
+    "block.integrateddynamics.crystalized_menril_brick": "門瑞歐結晶磚",
+    "block.integrateddynamics.block_menril_resin": "門瑞歐樹脂",
+    "block.integrateddynamics.drying_basin": "乾燥盆",
+    "block.integrateddynamics.drying_basin.info": "將物品和液體晾乾",
+    "block.integrateddynamics.squeezer": "擠壓架",
+    "block.integrateddynamics.squeezer.info": "站在上面反覆跳躍，利用重力將物品擠壓、壓碎或壓平。使用紅石信號來重置。",
+    "block.integrateddynamics.menril_torch": "門瑞歐火把",
+    "block.integrateddynamics.menril_torch_stone": "門瑞歐石柄火把",
+    "block.integrateddynamics.invisible_light": "不可見的光",
+    "block.integrateddynamics.menril_door": "門瑞歐木門",
+    "block.integrateddynamics.menril_planks_stairs": "門瑞歐木階梯",
+    "block.integrateddynamics.crystalized_menril_block_stairs": "門瑞歐結晶階梯",
+    "block.integrateddynamics.crystalized_menril_brick_stairs": "門瑞歐結晶磚階梯",
+    "block.integrateddynamics.delay": "延遲器",
+    "block.integrateddynamics.delay.info": "將變數前段時間的值公開作為項目。在合成台上清空。",
+    "block.integrateddynamics.block_liquid_chorus": "歌萊果漿",
+    "block.integrateddynamics.crystalized_chorus_block": "歌萊果漿結晶方塊",
+    "block.integrateddynamics.crystalized_chorus_brick": "歌萊果漿結晶磚",
+    "block.integrateddynamics.crystalized_chorus_block_stairs": "歌萊果漿結晶階梯",
+    "block.integrateddynamics.crystalized_chorus_brick_stairs": "歌萊果漿結晶磚階梯",
+    "block.integrateddynamics.mechanical_squeezer": "電動擠壓架",
+    "block.integrateddynamics.mechanical_squeezer.info": "利用電力將物品擠壓、壓碎或壓平",
+    "block.integrateddynamics.mechanical_drying_basin": "電動乾燥盆",
+    "block.integrateddynamics.mechanical_drying_basin.info": "用電力將物品和液體晾乾",
+    "block.integrateddynamics.crystalized_menril_block_slab": "結晶門瑞歐半磚",
+    "block.integrateddynamics.crystalized_menril_brick_slab": "結晶門瑞歐磚半磚",
+    "block.integrateddynamics.crystalized_chorus_block_slab": "歌萊果漿結晶半磚",
+    "block.integrateddynamics.crystalized_chorus_brick_slab": "歌萊果漿結晶磚半磚",
+
+    "_comment": "Items",
+    "item.integrateddynamics.general.id": "§e§lID：§r%s",
+    "item.integrateddynamics.wrench": "板手",
+    "item.integrateddynamics.wrench.info": "Shift + 右鍵以切換模式",
```

原文與譯文皆為按下**Shift**，但實際上模組是識別**潛行**，如此翻譯較能避免誤導。
```suggestion
    "item.integrateddynamics.wrench.info": "潛行 + 按下右鍵以切換模式",
```